mythril/laser/smt/bitvec.py

```diff
@@ -213,17 +240,49 @@ def UGT(a: BitVec, b: BitVec) -> Bool:
     :return:
     """
     annotations = a.annotations + b.annotations
+    if isinstance(a, BitVecFunc):
```

Comment:
A similar thing can be done here, like the BitVecFun right? like having an comparator and operator helper